community-manager-self-care.adoc

```diff
@@ -80,47 +80,47 @@ Set realistic expectations on deliverables and communicate frequently within the
 
-Believing that being in control is a reflection of our confidence in what we produce, is inaccurate.
+The belief that being "in control" is a reflection of our confidence in what we produce is inaccurate.
 It actually demonstrates that we are reluctant to show others that we are not perfect and struggle to accept anything less.
+Although maintaining this front can help shield us temporarily from others' emotions and criticisms towards us, it doesn't protect us from our own self-destruction caused by bottling them up.
 
-Although maintaining this front can help shield us temporarily from others' emotions and criticisms towards us, it doesn't protect ourselves from our own self-destruction caused by bottling them up.
+The consequences of suppressing your emotions will inevitably surface, and dramatically so.
+This is when our emotions _control us_ (and our actions), which often leads to rumination, collapse of relationships with one's team or community, and other negative outcomes like burnout.
 
-Consequences of suppressing your emotions is that they will inevitably surface and dramatically so.
-This is when our emotions control us and our actions, which often leads to rumination, break down of relationships with the team or community and other negative outcomes including burnout.
-
-The ultimate aim is to be "under control" in your leadership, emotions and workload.
-Our passion and attention is directed in a constructive and purposeful manner, this being especially important after when a mistake is made.
+The ultimate aim is to be "under control," both with your emotions and your workload.
+Here, our passion and attention are directed in a constructive and purposeful manner—a particularly important place to be in when we've made a mistake.
 
 The phrase "bring your whole self to work" is a good expression of allowing yourself this forgiveness.
-We feel we can admit when we don't know something, the right decision or simply acknowledging when we've made a mistake.
-But it also can be applied to recognizing and accepting that we can have good and bad days.
+We should feel like we can admit when we don't know something, haven't made the correct decision, or have made a mistake.
+And we should also recognize and accept that we can have good days and bad ones.
 
-Being honest with ourselves is as important as being honest with the community.
-Needing to admit that we may not be at our best and detracting yourself from unintentionally escalating  difficult situations into becoming worse situations would be the best choice to make at those times for the community.
+Being honest with ourselves is as important as being honest with our communities.
+Needing to admit that we may not be at our best and detracting yourself from unintentionally escalating difficult situations into becoming worse situations would be the best choice to make at those times for the community.
```

Comment:
This sentence reads a bit awkwardly to me, but I can't quite determine the point you're making, so I'm unsure how best to edit it. Can you help?

Reply:
I have made amendments to hopefully make it a bit more readable. Let me know if it requires further clarification!

Reply:
It does help, and I will perhaps offer another tweak or two, now that I see the direction. Thanks.